tests/testsuite/git.rs

```diff
@@ -4052,6 +4052,90 @@ src/lib.rs
         .run();
 }
 
+#[cargo_test(public_network_test)]
```

Comment:
You'll need `requires_git` as well

```suggestion
#[cargo_test(public_network_test, requires_git)]
```